### DISCUSSION TITLE
Raspi PI4 configure HDMI always on

### DISCUSSION THREAD

**Discussion by Libbei** [Screenly/Anthias#1984] [Q&A] [answered]
Hello. My Raspi PI4 doesn't turn on HDMI when it boots up. It's on a Samsung TV. How can I HDMI always on on it ? Please Help. Very Thanks. Philipp

**Comment by vpetersson** [comment, marked as answer]
The raspberry pi will not do this. You need to send a CEC command to the TV for it to wake up. There are tools available for this but not built into Anthias at this point.

**Comment by Libbei** [comment]
Thank you for your answer. The Raspi check on start the HDMI Ports. If no TV is connected, it switches the HDMI Port off. On my 5 years old Panasonic TV run is fine. I do the TV on als first and second the Raspi. On my new Samsung TV It does not work.

I must deactivate the HDMI Port check on Raspi. then it should probably work. but I do not know how

Thanks
